backend/engine/message_enum.py

```diff
@@ -0,0 +1,4 @@
+from enum import Enum
+
+# enums for determing types of messages
+MSG = Enum('MSG', 'chat tower_update tower_request game_update identifier info') # MSG_TYPES.chat = 1, etc.
```

Comment:
One of us needs to sync these message types with the message types from here:
https://github.com/zlailari/Towers/blob/master/public/js/Connection.js